src/loggerMiddleware.d.ts

```diff
@@ -0,0 +1,10 @@
+import Rodux from "@rbxts/rodux";
+
+interface LoggerMiddleware {
+    middleware: Rodux.Middleware;
+    outputFunction: (outputString: string) => void;
+}
+
+declare const LoggerMiddleware: LoggerMiddleware
+
+export = LoggerMiddleware
```

Comment:
You should add a trailing newline at the end of the file. Your editor will do this for you if you use its format feature.